runtime/reconcilers/project_parser.go

```diff
@@ -723,3 +731,11 @@ func equalAPISpec(a, b *runtimev1.APISpec) bool {
 func equalConnectorSpec(a, b *runtimev1.ConnectorSpec) bool {
 	return proto.Equal(a, b)
 }
+
+func keys(m map[string]any) []string {
```

Comment:
Use `maps.Keys`?

Reply:
We are not on go 1.23 yet.

Reply:
Just use `"golang.org/x/exp/maps"` instead then – already used various places in the code